openapi-codegen/core/src/main/scala/sttp/tapir/codegen/BasicGenerator.scala

```diff
@@ -169,10 +172,6 @@ object BasicGenerator {
       |      case Some(values) => DecodeResult.sequence(values.split(',').toSeq.map(e => support.rawDecode(List(e)))).map(r => Some(CommaSeparatedValues(r.toList)))
       |    }(_.map(_.values.map(support.encode).mkString(",")))
       |}
-      |implicit def makeExplodedQuerySeqCodecFromSupport[T](implicit support: QueryParamSupport[T]): sttp.tapir.Codec[List[String], ExplodedValues[T], sttp.tapir.CodecFormat.TextPlain] = {
```

Comment:
Since we now provide a `Codec[String, T, sttp.tapir.CodecFormat.TextPlain]`, from which we can derive a `Codec[List[String], List[T], sttp.tapir.CodecFormat.TextPlain]`, the final implicit derivation is provided by `makeExplodedQuerySeqCodecFromListSeq` and this is now redundant (it also causes explicit resolution conflicts in scala 3 if left in, although scala 2 seems happy enough with the ambiguity for some reason).